discord/role.py

```diff
@@ -94,13 +109,35 @@ def is_premium_subscriber(self) -> bool:
         return self._premium_subscriber is None
 
     def is_integration(self) -> bool:
-        """Whether the role is managed by an integration."""
+        """Whether the guild manages the role through some form of
+        integrations such as Twitch or trough guild subscriptions.
+        """
         return self.integration_id is not None
 
+    def is_available_for_purchase(self) -> bool:
+        """Whether the role is available for purchase.
+
+        Returns ``True`` if the role is available for purchase,
+        ``False`` if it is not or if the role is not linked to a guild subscription.
+
+        .. versionadded:: 2.7
+        """
+        return self._available_for_purchase is None
+
+    def is_guild_connections_role(self) -> bool:
+        """Whether the role is a guild connections role.
+
+        .. Versionadded:: 2.7
```

Comment:
causing docs build failure
```suggestion
        .. versionadded:: 2.7
```